spec/acceptance/itk_spec.rb

```diff
@@ -1,27 +1,20 @@
 require 'spec_helper_acceptance'
 
-case fact('osfamily')
-when 'Debian'
+case os[:family]
+when 'debian'
```

Comment:
Here you are changing it I think. `os[:family]` is `ubuntu` on Ubuntu where `fact('osfamily')` is `Debian`.

Reply:
hey @ekohl thank you for your patience, support and help. 
os[:family] is returning ubuntu -> you're right
 host_inventory['facter']['os']['name'] returns Ubuntu
 host_inventory['facter']['os']['family'] returns Debian -> what we need here
It's a bit confusing. Please correct me if i'm wrong

Reply:
You're right. I'd consider never using `os` and always using `host_inventory['facter']` (which I didn't know existed).